api/src/services/eml-service.ts

```diff
@@ -0,0 +1,380 @@
+import bbox from '@turf/bbox';
```

Comment:
Really nice and clean work on this file. I was just thinking to make the helper functions `private` methods so we dont expose those methods to initiators, will be bit cleaner while deciding what needs to be called. Also rearrage privates on top of class.

Reply:
Yeah I can look into that.  I bounce back and forth on private a bit, as it does make it nicer to use, but it can also make it more complicated to test, and since none of this code is used by external parties, it doesn't really matter if internal code is exposed.  If we were making an NPM package or something, then definitely you'd want to not expose certain pieces.